src/main/java/com/icthh/xm/ms/entity/domain/FunctionContext.java

```diff
@@ -2,6 +2,7 @@
 
 import com.fasterxml.jackson.annotation.JsonIdentityInfo;
 import com.fasterxml.jackson.annotation.JsonIdentityReference;
+import com.fasterxml.jackson.annotation.JsonIgnore;
```

Comment:
`@Table(name = "function_context")`

this commons can be used in miscroservices without database, FunctionContext looks like something wrong in commons, maybe better to remove it from commons and use in commons FunctionResult